MangaManager/src/MetadataManager/CoverManager/CoverManager.py

```diff
@@ -2,43 +2,74 @@
 import logging
 import platform
 import tkinter
+from idlelib.tooltip import Hovertip
 from os.path import abspath
 from tkinter import Frame, CENTER, Button, NW
 from tkinter import messagebox as mb
 from tkinter.filedialog import askopenfile
 from tkinter.ttk import Treeview
 
-from PIL import Image, ImageTk
+import numpy as np
+from PIL import Image, ImageTk, ImageChops
 from pkg_resources import resource_filename
 
 from src.Common.loadedcomicinfo import LoadedComicInfo, CoverActions
-from src.MetadataManager.GUI.widgets.CanvasCoverWidget import CoverFrame, CanvasCoverWidget
 from src.MetadataManager.GUI.scrolledframe import ScrolledFrame
 from src.MetadataManager.GUI.widgets import ButtonWidget
+from src.MetadataManager.GUI.widgets.CanvasCoverWidget import CoverFrame, CanvasCoverWidget
 from src.MetadataManager.MetadataManagerGUI import GUIApp
 from src.Settings.SettingHeading import SettingHeading
 from src.Settings.Settings import Settings
 
-action_template = abspath(resource_filename(__name__, '../../../res/cover_action_template.png'))
-
-
-def on_button_click(_, loaded_cinfo: LoadedComicInfo, front_or_back):
-    print("Clicked button.")
-    print(f"Is: {front_or_back}")
-    print(f"Path: {loaded_cinfo.file_path}")
-
+action_template = abspath(resource_filename(__name__, '../../../res/cover_action_template.png'))  # TODO: Use resource factory
```

Comment:
Don't use abspath, use ResourceLoader().get('cover_action_template.png'). This reduces pathing issues if code is moved and streamlines.

Reply:
This will be solved once it gets merged.